### PR TITLE
Common team details UI + usage in AWX

### PR DESCRIPTION
**Pull request by vidyanambiar**
Common component `frontend/common/access/TeamDetails.tsx` that can be used to create team details pages across services.

image

### CODE REVIEW

**Review by nixocio — APPROVED** [review]
LGTM

**Review by kialam — APPROVED** [review]
Pulled this down and tested locally. LGTM

File: cypress/fixtures/team.json [hunk 1]
```diff
@@ -33,6 +33,6 @@
   "created": "2023-02-10T15:08:32.547728Z",
   "modified": "2023-02-10T15:08:32.547752Z",
   "name": "Team 2 Org 0",
-  "description": "",
+  "description": "test",
```
Comment (AlexSCorey) [inline cypress/fixtures/team.json:36]: I like to be a bit more verbose with the values here so it is clear what is being rendered. I often use This is a description. for this value. Could make trouble shooting a bit easier in the future.

**Review by AlexSCorey — APPROVED** [review]
Just 1 suggestion.  Its nitpicky.  I'll approve and let you decide if you want to change it.

**Pull request merged**